src/TicTacToe.h

```diff
@@ -0,0 +1,29 @@
+#ifndef __TICTACTOE_H__
+#define __TICTACTOE_H__
+
+class TicTacToe
+{
+private:
+	char table[3][3];
+	char player[2];
+	int  p;
+
+public:
+	TicTacToe();
+	void print_Table();
+	void playVsPlayer();
+	void playVsAi();
+	void next_Player();
+	bool isNotValidPlace(int x, int y);
+	int isNotWin();
```

Comment:
It would be clearer if this method was named "hasWon" (it's more logical to check if someone won, than to check if not won).